api/v1/database_types.go

```diff
@@ -220,6 +224,39 @@ type ExtensionSpec struct {
 	Schema string `json:"schema,omitempty"`
 }
 
+// FDWSpec configures an Foreign Data Wrapper in a database
+type FDWSpec struct {
+	DatabaseObjectSpec `json:",inline"`
+
+	// Name of the handler function (e.g., "postgres_fdw_handler")
+	Handler string `json:"handler,omitempty"`
+
+	// Name of the validator function (e.g., "postgres_fdw_validator")
+	Validator string `json:"validator,omitempty"`
```

Comment:
I've tried the patch and tested several FDW implementations, and commonly, it is not explained what you should put in these fields. Usually, the FDW is created from the extension initialization; we should probably make these fields really optional by making them string pointers and not modifying the current value if they are set to nil.